eox_nelp/admin/course_creators.py

```diff
@@ -0,0 +1,36 @@
+"""courseCreators admin file.
```

Comment:
```suggestion
"""CourseCreators admin file.
```

Reply:
Resolved in https://github.com/eduNEXT/eox-nelp/pull/2/commits/a125a7569bbfd536f28897a1a2c94aa41e39a324